### ISSUE TITLE
could you please example code for  cnv plots

### ISSUE THREAD

**Issue by claus-h-g** [zhou-lab/sesame-tutorials#1]
Hello,
Many thanks for providing Sesame and the tutorial.
I did try to follow your vignette and the code provided with idats  generated on a human Epic V1 and tried to generate a CNV plot. I did follow you instructions on github SeSAMe Supplemental Vignette (https://zhou-lab.github.io/sesame/v1.18/supplemental.html). Currently I do not understand at all how you generate a file or  object you refer to "EPIC.1.SigDF".
Could you please provide an example code similar to the tutorial ?
Many thanks & best wishes
Claus

**Comment by zwdzwd** [comment]
Hi,
If you run `openSesame(..., func=NULL)` you will get SigDFs. Or `readIDATpairs` also generates that

**Comment by claus-h-g** [comment]
Many thanks for your fast reply.
I think I did solve the first step.
However I can not reproduce your code from the vignette with your example files on Ubuntu 20.04.
In a fresh installed R environment 4.3.0 (2023-04-21) I did install sesame:
> sesame_checkVersion()
SeSAMe requires matched versions of R, sesame, sesameData and ExperimentHub.
Here is the current versions installed:
R: 4.3.0
Bioconductor: 3.17
sesame: 1.18.4
sesameData: 1.18.0
ExperimentHub: 2.8.1

than I did install DNAcopy package with 

BiocManager::install("DNAcopy")

> library(sesame)
> sesameDataCache()
> sdf = sesameDataGet('EPIC.1.SigDF')
> segs <- cnSegmentation(sdf)
> visualizeSegments(segs)
Error in grid.Call(C_stringMetric, as.graphicsAnnot(x$label)) : 
  X11 font -adobe-helvetica-%s-%s-*-*-%d-*-*-*-*-*-*-*, face 1 at size 11 could not be loaded
> ?X11Fonts

this error is not remove by installing all txlive packages.
checking with:
 sudo apt install texlive*
prompts in:

0 upgraded, 0 newly installed, 0 to remove and 3 not upgraded.

How do I fix this?
